yarn.lock

```diff
@@ -6379,9 +6379,9 @@ typedoc@^0.26.0:
     yaml "^2.5.1"
 
 typescript@^5.3.3:
```

Comment:
Nitpick: it turns out that builds fail with `5.3.3` but pass with `5.4.2` or higher, so the `package.json` should use `^5.4.2`.